Model/updatePosts.php

```diff
@@ -0,0 +1,37 @@
+
+<?php
+
+session_start();
+require('../Model/DbConnection.php');
+$db = new DbConnection('postit');
+if (isset($_REQUEST['setPost']) && $_REQUEST['setPost'] == 'TRUE') {
+    $image = "";
+    $audio = "";
+    $video = "";
+    $comment = "";
```

Comment:
Initialising variables could have been done before the if block with proper comments.